autoload/ale/references.vim

```diff
@@ -67,19 +67,42 @@ function! ale#references#HandleTSServerResponse(conn_id, response) abort
 endfunction
 
 function! ale#references#FormatLSPResponseItem(response_item, options) abort
+    let l:line= a:response_item.range.start.line
+    let l:col = a:response_item.range.start.character
+    let l:filename = ale#util#ToResource(a:response_item.uri)
+
+    try
+        let l:line_text = readfile(l:filename)[l:line]
+    catch
+        " cannot read file, this happens in tests
```

Comment:
This is a workaround to the following test failures (which use a dummy filename)

`LSP reference responses should be put to quickfix` and `LSP reference responses should be handled`

here https://github.com/dense-analysis/ale/blob/add-preview-alefindreferences/test/test_find_references.vader

I'll try and come up with a better solution for tests.